gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -234,13 +234,6 @@ add_setup_done_file (void)
 static void
 done_cb (GtkButton *button, GisSummaryPage *page)
 {
-  gchar *file;
```

Comment:
We ended up a bit in a mixed state with this commit; you could either simply comment out/remove this code, or you should also remove the `gnome-welcome-tour` file itself. For sake of simplicity, I think I prefer the former.